packages/itwin/reports-config-widget/src/test/test-utils.tsx

```diff
@@ -4,35 +4,27 @@
 *--------------------------------------------------------------------------------------------*/
 import * as React from "react";
 import { render as rtlRender } from "@testing-library/react";
-import { ReportsApiConfigContext } from "../widget/context/ReportsApiConfigContext";
-import type { Store } from "redux";
-import { combineReducers, createStore } from "redux";
-import {
-  FrameworkReducer,
-  SyncUiEventDispatcher,
-  UiFramework,
-} from "@itwin/appui-react";
-import type { IModelConnection } from "@itwin/core-frontend";
-import { ReportsConfigWidget } from "../ReportsConfigWidget";
 import userEvent from "@testing-library/user-event";
-import { REPORTS_CONFIG_BASE_URL } from "../widget/ReportsConfigUiProvider";
+import type { ReportsConfigContextProps } from "../widget/components/ReportsConfigContext";
+import { ReportsConfigContext } from "../widget/components/ReportsConfigContext";
 
 export const mockAccessToken = async () => "Bearer eyJhbGci";
 
 interface WrapperProps {
   children?: React.ReactNode;
 }
+export const mockITwinId: string = "mockiTwinId";
 
-function render(ui: React.ReactElement, { ...options } = {}) {
+function render(ui: React.ReactElement, contextProps?: Partial<ReportsConfigContextProps>, { ...options } = {}) {
   const Wrapper = ({ children }: WrapperProps) => (
-    <ReportsApiConfigContext.Provider
-      value={{
-        getAccessToken: mockAccessToken,
-        baseUrl: REPORTS_CONFIG_BASE_URL,
-      }}
+    <ReportsConfigContext
+      getAccessToken={mockAccessToken}
+      iTwinId={contextProps?.iTwinId ?? mockITwinId}
+      bulkExtractor={contextProps?.bulkExtractor}
```

Comment:
I don't think you need to specify `bulkExtractor` explicitly here. The line below should take care of it.

Reply:
Yup, don't need it. Forgot to remove it. There used to be a default mock here.